syntax = "proto3";
package streetgangpb;


message CreateWorldRequest
{
	string world_name = 1;
}
